great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -83,18 +83,17 @@ def __init__(
         self._cloud_mode = True  # property needed for backward compatibility
 
         # Chetan - 20221208 - not formally deprecating these values until a future date
-        cloud_base_url = (
-            cloud_base_url if cloud_base_url is not None else ge_cloud_base_url
-        )
-        cloud_access_token = (
-            cloud_access_token
-            if cloud_access_token is not None
-            else ge_cloud_access_token
-        )
-        cloud_organization_id = (
-            cloud_organization_id
-            if cloud_organization_id is not None
-            else ge_cloud_organization_id
+        (
+            cloud_base_url,
+            cloud_access_token,
+            cloud_organization_id,
+        ) = self._resolve_CloudDataContext_cloud_args(
```

Comment:
I generally reference static methods with the name of the class (not self, since self isn't an argument)

Reply:
changed!